### ISSUE TITLE
UglifyJS throws exception

### ISSUE THREAD

**Issue by petermikitsh** [nashdot/accounting-js#17]
When including the `formatNumber` function in a production optimized webpack build, UglifyJs throws an error.

```js
// webpack.config.js

const CompressionPlugin = require("compression-webpack-plugin");
const webpack = require('webpack');

module.exports = {
  entry: [
    './node_modules/accounting-js/lib/formatNumber'
  ],
  output: {
    filename: '[name].js'
  },
  module: {
    rules: [
      {
        test: /\.js$/,
        loader: "babel-loader",
        exclude: /node_modules/
      }
    ]
  },
  plugins: [
    new CompressionPlugin({
      asset: "[path].gz[query]",
      algorithm: "gzip",
      test: /\.js$/
    })
  ]
};
```

```
> webpack -p --display verbose

Hash: 7b5a7e38df52dd5294ee
Version: webpack 3.0.0
Time: 126ms
     Asset     Size  Chunks             Chunk Names
   main.js  10.3 kB       0  [emitted]  main
main.js.gz  3.26 kB          [emitted]  
Entrypoint main = main.js
chunk    {0} main.js (main) 6.13 kB [entry] [rendered]
    > main [1] multi ./node_modules/accounting-js/lib/formatNumber 
    [0] ./node_modules/accounting-js/lib/settings.js 648 bytes {0} [depth 2] [built]
        [exports: default]
        [only some exports used: default]
        harmony import ./settings [2] ./node_modules/accounting-js/lib/formatNumber.js 4:0-34
        harmony import ./settings [5] ./node_modules/accounting-js/lib/toFixed.js 2:0-34
    [1] multi ./node_modules/accounting-js/lib/formatNumber 28 bytes {0} [depth 0] [built]
    [2] ./node_modules/accounting-js/lib/formatNumber.js 1.73 kB {0} [depth 1] [built]
        [exports: default]
        single entry ./node_modules/accounting-js/lib/formatNumber [1] multi ./node_modules/accounting-js/lib/formatNumber main:100000
    [3] ./node_modules/object-assign/index.js 2.11 kB {0} [depth 2] [built]
        [only some exports used: default]
        harmony import object-assign [2] ./node_modules/accounting-js/lib/formatNumber.js 1:0-41
    [4] ./node_modules/accounting-js/lib/internal/stripInsignificantZeros.js 322 bytes {0} [depth 2] [built]
        [exports: default]
        [only some exports used: default]
        harmony import ./internal/stripInsignificantZeros [2] ./node_modules/accounting-js/lib/formatNumber.js 3:0-74
    [5] ./node_modules/accounting-js/lib/toFixed.js 1.06 kB {0} [depth 2] [built]
        [exports: default]
        [only some exports used: default]
        harmony import ./toFixed [2] ./node_modules/accounting-js/lib/formatNumber.js 5:0-32
    [6] ./node_modules/accounting-js/lib/internal/checkPrecision.js 223 bytes {0} [depth 3] [built]
        [exports: default]
        [only some exports used: default]
        harmony import ./internal/checkPrecision [5] ./node_modules/accounting-js/lib/toFixed.js 1:0-56

ERROR in main.js from UglifyJs
Unexpected token operator «=», expected punc «,» [main.js:131,35]
npm ERR! code ELIFECYCLE
npm ERR! errno 2
```

**Comment by petermikitsh** [comment]
Identified the problem:

Default parameters are used in the method signature of `formatNumbers`, and that makes UglifyJS sad.

Found a solution:

1. Added the `es2015` preset to my `.babelrc` file:

```
// .babelrc
{
"presets": ["es2015"
}
```

2. Updated my js loader to include `accounting-js`:

```
  module: {
    rules: [
      {
        test: /\.js$/,
        loader: "babel-loader",
        exclude: /node_modules\/(?!accounting-js)/
      }
    ]
  },
```

Maybe the `.babelrc` file in this repo should be updated to include the `es2015` preset? I'll leave this open for feedback from maintainers.

**Comment by petermikitsh** [comment]
Actually, you probably just want to move `babel-preset-es2015-rollup` from `devDependencies` to `dependencies`.

**Comment by STAH** [comment]
Corrected by 1e963eafb8f50ac8f4c165ae782f587a4a41fedf
